_pages/01-applicants.md

```diff
@@ -25,51 +25,90 @@ For additional questions, please email <a href="mailto:oceanhkw@uw.edu" target="
+Oceanhackweek is not only a place to learn new skills but also a place to share expertise and build a community together.
+If you are already an expert but are interested in sharing and helping others,
+we welcome you to pitch project ideas,
+contribute to hacking,
+and assist other participants during the tutorials by being a teaching assistant!
+
+**Please mention this in the application. Send us an email if you have any questions!**
 
 **Q: I have strong computational skills but have not worked in oceanography?**
 
-To ensure that participants will be interested in the oceanography problems they are solving and can give back to the field later on, we require that participants have had at least some familiarity or prior experience with oceanography data.
+To ensure that participants will be interested in the oceanography problems they are solving and can give back to the field later on,
+we require that participants have had at least some familiarity or prior experience with oceanography data.
 
 **Q: I work in an ocean engineering/robotics/consulting company. Can I apply?**
 
-Make sure to explain your motivation for participation in the application form! We may hold a career panel discussion, so please mention if you are interested in participating.
+We expect participants from the private sector to pay for their own expenses.
+Make sure to explain your motivation for participation in the application form!
+We may hold a career panel discussion,
+so please mention if you are interested in participating.
 
 **Q: I work in a non-US institution / organization / company. Can I apply?**
 
 Yes, we welcome applications from abroad.
 
-**Q: I was a participant in a previous Oceanhackweek. Can I apply again?**
+**Q: I was a participant in previous Oceanhackweek. Can I apply again?**
+
+Yes, we welcome applications from previous participants,
+but priority will be given to new applicants.
+If you are interested in being a TA (teaching assistant) during the hackweek,
+please mention that in the application.
+
+**Q: Am I supposed to participate in all tutorials and presentations?**
 
-Yes, we welcome applications from previous participants, but priority will be given to new applicants. If you are interested in being a TA (teaching assistant) during the hackweek, please mention that in the application.
+Yes. While we understand that not all topics may be 100% relevant to you we want to maximize engagement. Note that we post the lessons and videos online later, so if you are interested in only a portion of the OHW consider that an option rather than taking someone spot.
```

Comment:
I would like some help making this one more "positive."

Reply:
I've reviewed everything else but will leave this one to @leewujung !